cypress/integration/mytools.js

```diff
@@ -144,8 +144,11 @@ describe('Dockstore my tools', function() {
                 .get('#deregisterConfirmButton')
                 .click()
 
-            
-            // This should be activated later when https://github.com/ga4gh/dockstore/issues/1318 is resolved
+            // This is deactivated because:
```

Comment:
Maybe add what can be done in the future to resolve this then (at which time this will be uncommented)
If this is likely never to be uncommented, just delete it